tests/AtlasSearchTest.php

```diff
@@ -193,10 +194,13 @@ public function testDatabaseBuilderAutocomplete()
 
         self::assertInstanceOf(LaravelCollection::class, $results);
         self::assertCount(3, $results);
+        // Sort results, because order is not guaranteed
```

Comment:
This change looks unrelated to the Scout work. Should it instead of associated with [PHPORM-276](https://jira.mongodb.org/browse/PHPORM-276)?

Reply:
This fixes the test.